<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComparisonSelectionAdvancedWidget</class>
 <widget class="QFrame" name="ComparisonSelectionAdvancedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>159</height>
   </rect>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <layout class="QGridLayout" name="widgetLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select Attribute Matrix</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DataArrayPathSelectionWidget" name="m_SelectedAttributeMatrixPath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="iconSize">
        <size>
         <width>2</width>
         <height>2</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QLabel
{
font: 10pt &quot;Arial&quot;;
font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Attribute Arrays to Threshold</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="ComparisonSetWidget" name="comparisonSetWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataArrayPathSelectionWidget</class>
   <extends>QToolButton</extends>
   <header>DataArrayPathSelectionWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ComparisonSetWidget</class>
   <extends>QWidget</extends>
   <header>ComparisonSetWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../Resources/SIMPL.qrc"/>
 </resources>
 <connections/>
</ui>
